apps/emqx/test/emqx_sys_mon_SUITE.erl

```diff
@@ -151,3 +166,8 @@ some_function(Parent, _Arg2) ->
         stop ->
             ok
     end.
+
+dead_port() ->
+    Port = erlang:open_port({spawn, "ls"}, []),
+    catch exit(Port, kill),
```

Comment:
just curious, why do we need `catch` here?

Reply:
I think this may not be needed, actually.  The test was crashing initially and I wrongly assumed it was this exit. That was fixed by trapping exits, and I forgot to remove the `catch`.